databricks/koalas/tests/test_dataframe_conversion.py

```diff
@@ -181,7 +181,9 @@ def test_to_latex(self):
         self.assert_eq(kdf.to_latex(sparsify=False), pdf.to_latex(sparsify=False))
         self.assert_eq(kdf.to_latex(index_names=False), pdf.to_latex(index_names=False))
         self.assert_eq(kdf.to_latex(bold_rows=True), pdf.to_latex(bold_rows=True))
-        self.assert_eq(kdf.to_latex(encoding='ascii'), pdf.to_latex(encoding='ascii'))
+        # TODO: error in pandas >= 1.0.0:
+        # `ValueError: buf is not a file name and encoding is specified.`
+        # self.assert_eq(kdf.to_latex(encoding='ascii'), pdf.to_latex(encoding='ascii'))
```

Comment:
Can we fix this case to match with pandas 1.0.0?

Reply:
If pandas 1.0.0 behaviour looks like a bug, we don't need to add `TODO` here.

Reply:
yeah, this rule has changed from pandas 1.0.0 like the below.
(refered from https://pandas.pydata.org/docs/whatsnew/v1.0.0.html)

> Added encoding argument to DataFrame.to_string() for non-ascii text (GH28766)

we can just change to do this will be tested only when pandas>=1.0.0